### PR TITLE
retry more often to avoid random errors

### PR DESCRIPTION
**Pull request by grosser**
/cc @zendesk/samson @zendesk/compute

### CODE REVIEW

**Review by hgokavarapuz — APPROVED** [review]

File: plugins/kubernetes/lib/samson_kubernetes/samson_plugin.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def self.connection_errors
   end
 
   def self.retry_on_connection_errors(&block)
-    Samson::Retry.with_retries connection_errors, 3, only_if: NOT_A_404, &block
+    Samson::Retry.with_retries connection_errors, 5, only_if: NOT_A_404, wait_time: 0.1, &block
```
Comment (uthark) [inline plugins/kubernetes/lib/samson_kubernetes/samson_plugin.rb]: is 0.1 is enough?
what about exponential backoff?

File: plugins/kubernetes/test/controllers/kubernetes/namespaces_controller_test.rb [hunk 2]
```diff
@@ -135,7 +135,7 @@
 
       it "shows creation errors" do
         assert_request(:get, "http://foobar.server/api/v1/namespaces/test", to_return: {status: 404}) do
-          assert_request(:post, "http://foobar.server/api/v1/namespaces", to_timeout: [], times: 4) do
+          assert_request(:post, "http://foobar.server/api/v1/namespaces", to_timeout: [], times: 6) do
```
Comment (uthark) [inline plugins/kubernetes/test/controllers/kubernetes/namespaces_controller_test.rb]: I'd extract 5 and 6 to constants to improve clarity and to reduce further changes, if we want to change from 5 to 10.

**Comment by grosser** [comment]
0.1 for now until we run into more trouble

File: plugins/kubernetes/test/controllers/kubernetes/namespaces_controller_test.rb (hunk 2 above)
Comment (grosser) [reply plugins/kubernetes/test/controllers/kubernetes/namespaces_controller_test.rb]: https://github.com/zendesk/samson/pull/3395

**Pull request merged**